doc/JSON_INFO.md

```diff
@@ -2483,7 +2483,9 @@ at level `2` to the item.
       "sound_variant": "bear_trap",
       "remove_trap": true,
       "spawn_items": [ "beartrap" ]
-    }
+    },
+    "trigger_message_u": "A bear trap closes on your foot!",  // This message will be printed when player steps on a trap
```

Comment:
I know it is a bit late to review this but what if the player has no feet? I am mainly thinking of this gastropod thing or even the player has paws instead of feet.

Reply:
The message is the same no matter which limbs and how many of them a creature has. 

We could add additional checks for limbs. Or simply rewrite the message.